var/spack/repos/builtin/packages/ior/package.py

```diff
@@ -17,6 +17,9 @@ class Ior(AutotoolsPackage):
     version('3.2.0', sha256='91a766fb9c34b5780705d0997b71b236a1120da46652763ba11d9a8c44251852')
     version('3.0.1', sha256='0cbefbcdb02fb13ba364e102f9e7cc2dcf761698533dac25de446a3a3e81390d')
 
+    version('3.3.0rc1', sha256='0e42ebf5b5adae60625bf97989c8e2519d41ea2e3d18561d7d5b945625317aa5')
+    version('develop', git='https://github.com/hpc/ior.git', branch='master')
```

Comment:
Can you sort these versions from newest to oldest? Do you think we should set the preferred version as 3.2.1 until a stable release of 3.3.0 comes out?

Reply:
I have reordered entries and set 3.2.1 as preferred.

I am not sure which one to keep preferred, on one side 3.2.1 is officially stable on the other hand it does not work with hdf5 if newer version of hdf5 or compilers are used. I think 3.2.1 used to work at some point. I see why it is not compiling (there is no cast to proper structure, so requested structure member is not visible). 3.3.0 also has slightly different output.

Anyway i set 3.2.1 as prefered.